PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureArtifactComboBox.java

```diff
@@ -64,7 +65,7 @@ public void setArtifact(@Nullable final AzureArtifact azureArtifact) {
     )
     protected List<? extends AzureArtifact> loadItems() throws Exception {
         final List<AzureArtifact> collect = fileArtifactOnly ?
-                new ArrayList<>() : AzureArtifactManager.getInstance(project).getAllSupportedAzureArtifacts();
+                new ArrayList<>() : ReadAction.compute(() -> AzureArtifactManager.getInstance(project).getAllSupportedAzureArtifacts());
```

Comment:
this fix can be reverted as I have fixed it at "root":
https://github.com/microsoft/azure-tools-for-java/commit/ff32305bc9387ca29c1f805401ff4923d16a6f01